docs/INSTALL.md

```diff
@@ -109,7 +112,7 @@ Assuming that Postgres 9.5 (or later) is installed:
 * Create the component databases:
 
   ```bash
-  for i in account device mediaapi syncapi roomserver signingkeyserver federationsender appservice e2ekey naffka; do
+  for i in account device mediaapi syncapi roomserver signingkeyserver federationsender appservice e2ekey naffka userapi_accounts userapi_devices keyserver ; do
```

Comment:
We already have `account`, `device` and `signingkeyserver` here. Is what you are saying with this change that there’s a sample config somewhere that disagrees with these instructions?

Reply:
Interesting, I didn't notice! 
It seems like the default config is looking for 
dendrite_userapi_accounts, dendrite_userapi_devices and dendrite_keyserver 
instead of 
dendrite_account, dendrite_device, and dendrite_ signingkeyserver.

Reply:
I think it isn't mentioned in particular ( at least I didn't find docs for switching to postgresql ).

I basically used the naming scheme of the sqlite-databases:

```
# Configuration for the Key Server (for end-to-end encryption).
key_server:
  internal_api:
    listen: http://localhost:7779
    connect: http://localhost:7779
  database:
    connection_string: file:keyserver.db
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
```
to
```
# Configuration for the Key Server (for end-to-end encryption).
key_server:
  internal_api:
    listen: http://localhost:7779
    connect: http://localhost:7779
  database:
    connection_string: postgresql://dendrite:<password>@localhost/dendrite_keyserver
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
```
___

```
# Configuration for the User API.
user_api:
  internal_api:
    listen: http://localhost:7781
    connect: http://localhost:7781
  account_database:
    connection_string: file:userapi_accounts.db
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
  device_database:
    connection_string: file:userapi_devices.db
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
```

to

```
# Configuration for the User API.
user_api:
  internal_api:
    listen: http://localhost:7781
    connect: http://localhost:7781
  account_database:
    connection_string: postgresql://dendrite:<password>@localhost/dendrite_userapi_accounts
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
  device_database:
    connection_string: postgresql://dendrite:<password>@localhost/dendrite_userapi_devices
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
```
___

```
# Configuration for the Signing Key Server (for server signing keys).
signing_key_server:
  internal_api:
    listen: http://localhost:7780
    connect: http://localhost:7780
  database:
    connection_string: file:signingkeyserver.db
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
```
to

```
# Configuration for the Signing Key Server (for server signing keys).
signing_key_server:
  internal_api:
    listen: http://localhost:7780
    connect: http://localhost:7780
  database:
    connection_string: postgresql://dendrite:<password>@localhost/dendrite_signingkeyserver
    max_open_conns: 100
    max_idle_conns: 2
    conn_max_lifetime: -1
```